app/models/wallet-connect-v2-client/wallet-connect-v2-client.ts

```diff
@@ -0,0 +1,338 @@
+    //   self.connect({ session })
+
+    //   self.listenToRequests()
+    // },
+    approveSessionRequest: flow(function * (proposal: SignClientTypes.EventArguments['session_proposal']) {
+      // Get required proposal data
+      const { id, params } = proposal
+      const { requiredNamespaces, relays } = params
+      const chainId = requiredNamespaces.cosmos?.chains[0];
+      // TODO: remove hacky check
+      // parse requiredNamespaces and check chain id and methods
+      if (!chainId.includes('likecoin')) {
+        return self.connector.reject({
+          id,
+          reason: getSdkError('UNSUPPORTED_CHAINS')
+        })
+      }
+      const { bech32Address }: { bech32Address: string } = yield self.env.authCoreAPI.getWalletConnectGetKeyResponse(
+        chainId.replace('cosmos:', ''),
+        { name: self.currentUser.likerID }
+      )
+      const accounts = [bech32Address].map((address) => `${chainId}:${address}`)
+      yield self.connector.approve({
+        id,
+        relayProtocol: relays[0].protocol,
+        namespaces: {
+          cosmos: {
+            accounts,
+            methods: ['cosmos_signAmino', 'cosmos_signDirect', 'cosmos_getAccounts'],
+            events: [],
+            chains: [chainId],
+          },
+        }
+      })
+      self.sessions = self.connector.session.getAll();
+
+      // self.serializedSession = JSON.stringify(self.connector.session)
+    }),
+  }))
+  .actions(self => ({
+    handleNewSessionRequest(
+      proposal: SignClientTypes.EventArguments['session_proposal']
+    ) {
+      const payload = {
+        method: 'session_proposal',
+        ...proposal,
+      }
+      // if (!self.isInAppBrowser) {
+        // Show WalletConnect Modal for loading UX
+        self.navigationStore.navigateTo({
+          routeName: 'App',
+          action: NavigationActions.navigate({
+            routeName: 'WalletConnect',
+            params: {
+              peerMeta: proposal.params.proposer.metadata,
+              payload,
+            },
+          }),
+        })
+      // }
+
+      // Approve session request directly without user's interaction
+      // self.approveSessionRequest(proposal)
+    },
+  }))
+  .actions(self => ({
+    approveRequest: flow(function * (payload: any, topic: string) {
+      if (isSessionRequest(payload.method)) {
+        self.approveSessionRequest(payload)
+      } else {
+        yield self.handleCallRequestApproval(payload, topic)
+      }
+    }),
+  }))
+  .actions(self => ({
+    listenToRequests() {
+      self.connector.on('session_proposal', self.handleNewSessionRequest)
+      self.connector.on('session_request', self.handleNewCallRequest)
+      self.connector.on('session_delete', self.handleDisconnect)
+    },
+  }))
+  .actions(self => ({
+    createClient: flow(function * (){
+      const client = yield SignClient.init({
+        projectId: self.getConfig('WALLET_CONNECT_PROJECT_ID', 'd36730c770de6ccc7db8591473e97f6f'),
```

Comment:
Should remove default value?
```suggestion
        projectId: self.getConfig('WALLET_CONNECT_PROJECT_ID'),
```

Reply:
Need some help on why `WALLET_CONNECT_PROJECT_ID` is not get from env here, other comments are updated

Reply:
I can see it
<img width="478" alt="image" src="https://github.com/likecoin/likecoin-app/assets/17264716/6b830e77-fd56-4243-bb5e-bb902408b4f9">